src/types.ts

```diff
@@ -80,3 +80,10 @@ export interface VolumeDisplayOptions {
 
 export const isOrthographicCamera = (def: Camera): def is OrthographicCamera =>
   def && (def as OrthographicCamera).isOrthographicCamera;
+
+export const enum ViewportCorner {
+  TOP_LEFT = 0,
```

Comment:
I'm pretty sure you can use string values! This is what I thought you meant to do all along:
TOP_LEFT = "top_left"